cli/cmd/proxy/list/command.go

```diff
@@ -192,7 +192,7 @@ func (c *ListCommand) fetchPods() ([]v1.Pod, error) {
 func (c *ListCommand) output(pods []v1.Pod) {
 	if c.flagAllNamespaces {
 		c.UI.Output("Namespace: All Namespaces\n")
-	} else {
+	} else if c.namespace != "" {
```

Comment:
This just avoids the problem of printing "Namespace :" if there is no namespace set.